ql/src/java/org/apache/hadoop/hive/ql/io/arrow/Serializer.java

```diff
@@ -97,7 +97,7 @@ class Serializer {
   private final VectorAssignRow vectorAssignRow;
   private int batchSize;
 
-  private final NullableMapVector rootVector;
+  private final StructVector rootVector;
```

Comment:
in the original master PR I can see
```NullableMapVector -> NonNullableStructVector
```
here:
```
NullableMapVector -> StructVector
```

Reply:
@abstractdog I think I have backported it correctly. If you see in the Serializer.java in the original commit also they have used StructVector in the first 2 occurrences and then the NonNullableStructVector in the last 2 occurrences in the Serializer class.

Reply:
In the original PR :
`
NullableMapVector -> StructVector
`
and
`
MapVector -> NonNullableStructVector
`

@abstractdog Please note that the import statements are misplaced in the original PR. instead of import x,y its import y,x